MentoringHomeWork/HW6/HW6.Task2/Program.cs

```diff
@@ -0,0 +1,6 @@
+﻿// See https://aka.ms/new-console-template for more information
+
+using HW6.Task2;
+
+NewArrayPosition ar = new NewArrayPosition();
+NewArrayPosition.AddNewPosition();
```

Comment:
The same - you created an object and don't use it - just call the static method.

Reply:
resolved